src/controllers/EventsController.php

```diff
@@ -499,12 +499,12 @@ public function putAction($request, $db)
             }
             $cfp_start_date = $request->getParameter("cfp_start_date", false);
             if (false !== $cfp_start_date && strtotime($cfp_start_date)) {
-                $cfp_start_date = new DateTime(strtotime($cfp_start_date), $tz);
+                $cfp_start_date = new DateTime('@' . strtotime($cfp_start_date), $tz);
```

Comment:
I'm nearly sure that the correct solution is to remove the `strtotime` as `DateTime`'s constructor has that functionality built in.